versions/2.0/vipps-login-migrate-api-1.0-to-2.0.md

```diff
@@ -0,0 +1,28 @@
+# Vipps Login API
+
+Document version 1.0.0.
+
+Reference:\
+[vipps-login-api-version-1.0](../../vipps-login-api)\
+[vipps-login-api-version-2.0](vipps-login-api.md)
+
+## Table of contents
+* [Overview](#overview)
+* [Convert to vipps login api version 2.0](#convert-to-vipps-login-api-version-20)
+
+## Overview:
+In vipps login api version 2.0 we are changing format of scope 'nnin' to 'nin'.
+We have also have a major change to the response from the '/userinfo' endpoint.
+These are changes to the format of claim 'birthdate' from dd.mm.yyyy to yyyy-mm-dd.
+Claim 'address' have been converte from list of all addresses to the users preferred address, 
+and the none default addresses are served under 'other_address'.
+
+## Convert to vipps login api version 2.0
+Until we can fully switch to vipps login api version 2.0 we need every merchant to additionally send inn scope 'api_version_2'.
```

Comment:
Suggested change: To control witch response to serve we need the merchant to send inn scope 'api_version_2'.

Reply:
Done.